src/CheckoutSdk/Instruments/InstrumentDetailsResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Checkout.Instruments
+{
+    public class InstrumentDetailsResponse : InstrumentDetails
+    {
+        public InstrumentCustomerResponse InstrumentCustomerResponse { get; set; }
```

Comment:
The name of this property is `customer` in the payload, so I don't think that this name gonna work.